release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -646,16 +646,17 @@ Some features in this release are currently in Technology Preview. These experim
 
 link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
 
-In the table below, features are marked with the following statuses:
+In the following tables, features are marked with the following statuses:
 
 * *TP*: _Technology Preview_
 * *GA*: _General Availability_
 * *-*: _Not Available_
 * *DEP*: _Deprecated_
 
-//TODO: remove anything that has been GA since 4.9?
+[id="ocp-networking-technology-preview-features"]
+=== Networking technology preview features
```

Comment:
Do you think that these headings should be discrete so that they don't show up in the TOC? At first, I thought they shouldn't show in the TOC (to be consistent with how it is now), but maybe it would actually be beneficial to have them. Just bringing it up as a thought just in case :)

Reply:
Yes! I think discrete headings would be good!